<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Slider3Ranges</class>
 <widget class="QWidget" name="Slider3Ranges">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>100</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>100</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="horizontalSpacing">
    <number>6</number>
   </property>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="labelFeatureName">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>25</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	font-weight: bold;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="2" colspan="2">
    <widget class="QCheckBox" name="checkBoxActionability">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>25</height>
      </size>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="text">
      <string>actionable</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QDoubleSpinBox" name="doubleSpinBoxMinimum">
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>25</height>
      </size>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="decimals">
      <number>1</number>
     </property>
     <property name="maximum">
      <double>1000000.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.500000000000000</double>
     </property>
    </widget>
   </item>
   <item row="1" column="1" rowspan="2" colspan="2">
    <widget class="QWidget" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>280</width>
       <height>50</height>
      </size>
     </property>
     <widget class="RangeValue" name="labelRangeMaximumValue">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>30</y>
        <width>21</width>
        <height>25</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="RangeValue" name="labelRangeMinimumValue">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>30</y>
        <width>21</width>
        <height>25</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="Range" name="labelRangeValue">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>0</y>
        <width>10</width>
        <height>25</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>10</width>
        <height>25</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>10</width>
        <height>25</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
	background-color: #049DD9;
	border-style:solid;
	border-width:1px;
	border-color: #049DD9;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="Range" name="labelRangeMaximum">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>0</y>
        <width>10</width>
        <height>15</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>10</width>
        <height>15</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>10</width>
        <height>15</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
	background-color: #376b80;
	border-style:solid;
	border-width:1px;
	border-color: #376b80;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="RangeValue" name="labelRangeValueValue">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>30</y>
        <width>21</width>
        <height>25</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="Slider" name="labelSlider">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>280</width>
        <height>5</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>280</width>
        <height>5</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>290</width>
        <height>5</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
	border-style:solid;
	border-width:1px;
	border-color: #049DD9;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="Range" name="labelRangeMinimum">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>10</width>
        <height>15</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>10</width>
        <height>15</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>10</width>
        <height>15</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
	background-color: #376b80;
	border-style:solid;
	border-width:1px;
	border-color: #376b80;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <zorder>labelRangeMaximumValue</zorder>
     <zorder>labelRangeMinimumValue</zorder>
     <zorder>labelRangeValueValue</zorder>
     <zorder>labelSlider</zorder>
     <zorder>labelRangeMinimum</zorder>
     <zorder>labelRangeMaximum</zorder>
     <zorder>labelRangeValue</zorder>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QDoubleSpinBox" name="doubleSpinBoxMaximum">
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>25</height>
      </size>
     </property>
     <property name="decimals">
      <number>1</number>
     </property>
     <property name="maximum">
      <double>1000000.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.500000000000000</double>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="labelAdjustPosition1">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>25</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	font-weight: bold;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QLabel" name="labelAdjustPosition2">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>25</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	font-weight: bold;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Range</class>
   <extends>QLabel</extends>
   <header>.Range.h</header>
  </customwidget>
  <customwidget>
   <class>Slider</class>
   <extends>QLabel</extends>
   <header>.Slider.h</header>
  </customwidget>
  <customwidget>
   <class>RangeValue</class>
   <extends>QLabel</extends>
   <header>.RangeValue.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
